### PR TITLE
chore: update contributing guidelines

### PR DESCRIPTION
**Pull request by lmiller1990**
Update the contributing guidelines to include some information about what kind of tests to add, and the minimum requirement of two :heavy_check_mark: before merging code.

### CODE REVIEW

**Review by rachelruderman — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -535,16 +535,31 @@ Independent packages are automatically released when code is merged into `master
...
 
+### Write Some Tests
+
+If you are adding a new feature or fixing a regression, ensure you add tests for it. Broadly speaking, there are three categories of tests you might consider:
+
+1. System Tests. Those go in the [`system-tests`](https://github.com/cypress-io/cypress/tree/develop/system-tests) directory. The README explains how they work. These are the slowest to run, so you generally only want to add a system-test if it's absolutely required (but don't let that discourage you; they are also the most realistic way to test Cypress). These are between System Tests and Unit Tests when it comes to speed of execution.
+2. E2E/Integration tests. Those are inside `cypress/e2e`, if the package has them. 
+3. Unit test. Those are inside of `test/unit`, if the package has them. These are the fastest and cheapest to execute.
```
Comment (warrensplayer) [inline CONTRIBUTING.md]: Did you intend the sentence "These are between System Tests and Unit Tests when it comes to speed of execution." to be at the end of bullet number 2 for "E2E/Integration tests." ?

Comment (BlueWinds) [inline CONTRIBUTING.md]:
```suggestion
1. Unit test. Those are inside of `test/unit`, if the package has them. These are the fastest and cheapest to execute.
2. E2E/Integration tests. Those are inside `cypress/e2e`, if the package has them. These are between Unit Tests and System Tests when it comes to speed of execution.
3. System Tests. Those go in the [`system-tests`](https://github.com/cypress-io/cypress/tree/develop/system-tests) directory. The README explains how they work. These are the slowest to run, so you generally only want to add a system-test if it's absolutely required (but don't let that discourage you; they are also the most realistic way to test Cypress).
```

Comment (BlueWinds) [reply CONTRIBUTING.md]: Reversed the order to put fast ones up top (especially since system tests says "prefer the other types").

**Review by BlueWinds — APPROVED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (lmiller1990) [reply CONTRIBUTING.md]: I did, fixed!
